openupgrade_scripts/apriori.py

```diff
@@ -5,21 +5,56 @@
 # Renamed modules is a mapping from old module name to new module name
 renamed_modules = {
     # odoo
+    "coupon": "loyalty",
+    "pos_coupon": "pos_loyalty",
+    "sale_coupon": "sale_loyalty",
+    "sale_coupon_delivery": "sale_loyalty_delivery",
+    "website_sale_coupon": "website_sale_loyalty",
+    "website_sale_coupon_delivery": "website_sale_loyalty_delivery",
+    # odoo/enterprise
+    "helpdesk_sale_coupon": "helpdesk_sale_loyalty",
+    "sale_coupon_taxcloud": "sale_loyalty_taxcloud",
+    "sale_coupon_taxcloud_delivery": "sale_loyalty_taxcloud_delivery",
     # OCA/...
 }
 
 # Merged modules contain a mapping from old module names to other,
 # preexisting module names
 merged_modules = {
     # odoo
+    "fetchmail": "mail",
+    "gift_card ": "loyalty",
+    "pos_gift_card": "pos_loyalty",
+    "sale_gift_card": "sale_loyalty",
+    "website_sale_gift_card": "website_sale_loyalty",
     # OCA/...
 }
 
 # only used here for upgrade_analysis
 renamed_models = {
     # odoo
+    "account.analytic.group": "account.analytic.plan",
+    "account.tax.carryover.line": "account.report.external.value",
+    "account.tax.report": "account.report",
+    "account.tax.report.line": "account.report.line",
+    "coupon.coupon": "loyalty.card",
+    "coupon.program": "loyalty.program",
+    "coupon.reward": "loyalty.reward",
+    "coupon.rule": "loyalty.rule",
+    "mail.channel.partner": "mail.channel.member",
+    "payment.acquirer": "payment.provider",
+    "payment.acquirer.onboarding.wizard ": "payment.provider.onboarding.wizard",
+    "sale.coupon.apply.code": "sale.loyalty.coupon.wizard",
+    "sale.payment.acquirer.onboarding.wizard": "sale.payment.provider.onboarding.wizard",
+    "stock.location.route": "stock.route",
+    "stock.production.lot": "stock.lot",
     # OCA/...
 }
 
 # only used here for upgrade_analysis
-merged_models = {}
+merged_models = {
+    # odoo
+    "account.account.type": "account.account",
```

Comment:
I don't think this can be considered a merge...

Reply:
indeed, in a sense, but here merged models is used by the module_analysis to understand if fields are new or not or moved on a model. writing that will generate potential bad analysis on ``account.account`` model I think.